components/molecules/LogoOONIRun.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { FC } from 'react'
```

Comment:
Need to test if this will work properly or not but this thing is only being imported in `ooni/run`